docs/source/usersguide/input.rst

```diff
@@ -247,23 +198,57 @@ problem. It has the following attributes/sub-elements:
 
     *Default*: None
 
-``<fixed_source>`` Element
+``<generations_per_batch>`` Element
+-----------------------------------
+
+The ``<generations_per_batch>`` element indicates the number of total fission
+source iterations per batch for an eigenvalue calculation. This element is
+ignored for all run modes other than "eigenvalue".
+
+  *Default*: 1
+
+``<inactive>`` Element
+----------------------
+
+The ``<inactive>`` element indicates the number of inactive batches used in a
+k-eigenvalue calculation. In general, the starting fission source iterations in
+an eigenvalue calculation can not be used to contribute to tallies since the
+fission source distribution and eigenvalue are generally not converged
+immediately.
```

Comment:
Note that this is ignored for all run modes other than "eigenvalue" exactly like `<generations_per_batch>`